core/arch/arm/kernel/sub.mk

```diff
@@ -36,6 +36,7 @@ srcs-$(CFG_ARM64_core) += misc_a64.S
 srcs-y += mutex.c
 srcs-y += wait_queue.c
 srcs-$(CFG_PM_STUBS) += pm_stubs.c
+cflags-pm_stubs.c-y += -Wno-suggest-attribute=noreturn
```

Comment:
What about for instance `core/arch/arm/plat-vexpress/main.c` when compiling for QEMU Virt?